### PR TITLE
research groups added to filters, fixed css on blogs.index

### LINKED ISSUE #108: As a User I want to be able to select research groups on the filter and blogs so that I can see them
Acceptance Criteria:
GIVEN: you are on the Admin panel.
WHEN: you add a new research group.
THEN: the added research group should be visible on the blogs and filter.

### PR DESCRIPTION
**Pull request by awqfrxx**
koen and quinten added:
filter fixes, css styling on cards and blogs
title is clickable now on blogs.index
dynamically added research groups into filters and blogs when created

### CODE REVIEW

**Review by Sanyi0001 — APPROVED** [review]
Currently I only have time to check the result, but for me it is working fine

File: public/css/app.css [hunk 1]
```diff
@@ -123,19 +123,67 @@
...
+    padding: 2%;
+    margin-bottom: 0.2em;
+}
+
+.card_content-half {
+    background-color: white;
+    border-radius: 10px;
+    padding: 4%;
+    margin-bottom: 0.2em;
+    width: 29.4em;
```
Comment (Koreyoshi159) [inline public/css/app.css]: 29.4 ?

**Review by avanheeren — APPROVED** [review]
nice work

File: public/css/app.css (hunk 1 above)
Comment (awqfrxx) [reply public/css/app.css]: 29.4, but how does it look?

**Review by koenhamstra — APPROVED** [review]
works fine for me

File: public/css/app.css (hunk 1 above)
Comment (AlexGit123) [reply public/css/app.css]: good to me

**Review by AlexGit123 — APPROVED** [review]

**Pull request merged**